README.md

```diff
@@ -1,4 +1,8 @@
-#Welcome to WeBWorK
+![main workflow](https://github.com/pstaabp/pg/actions/workflows/coverage.yml/badge.svg)
+[![codecov](https://codecov.io/gh/pstaabp/pg/branch/unit-test/graph/badge.svg?token=H7WYHBDB9S)](https://codecov.io/gh/pstaabp/pg)
+![GitHub last commit](https://img.shields.io/github/last-commit/pstaabp/pg/unit-test)
+
+# Welcome to WeBWorK
```

Comment:
The `#Welcome to WeBWorK` line should be the first line in this file.  The badges should come after that.  The first line should be a top-level heading.  markdownlint complains about this.  Of course markdownlint complains about a lot of other things in this file too, but this is the only one added by this pull request.  You do fix one issue below!